coxeter/shape_classes/base_classes.py

```diff
@@ -28,6 +28,38 @@ def gsd_shape_spec(self):
         """dict: Get a :ref:`complete GSD specification <shapes>`."""  # noqa: D401
         return {}
 
+    def compute_form_factor_amplitude(self, q):
+        r"""Calculate the form factor intensity.
+
+        In solid state physics, scattering theory is concerned with understanding the
+        ways in which light is scattered from a sample. For a single point particle at
+        position P, the the amplitude of a scattered wave observed at position Q is the
+        product of the incoming wave amplitude (which follows the standard equation for
+        a traveling wave) and the scattering density at P. For a crystal composed of
+        many point particles, the intensity of the resulting superposition of waves can
+        be identified as the Fourier transform of the total scattering density. When the
+        particles are not point particles, the scattering density of the particles in
+        their local coordinate systems are no longer identical. Conveniently, this
+        component is separable in the Fourier transform of the total density; as a
+        result, the scattering scattering intensity can be decomposed into two terms,
+        the Fourier transform of the distribution of scatterers and the Fourier
+        transform of each scatterer in its local coordinate system. The first term is
+        known as the static structure factor :math:`S(\vec{q})` and describes the
+        spatial distribution of scatterers, while the second term is called the form
+        factor :math:`F(\vec{q})` and describes the local scattering profile.
```

Comment:
Typically lowercase f. Also would love to have Wikipedia links in this docstring.
```suggestion
        factor :math:`f(\vec{q})` and describes the local scattering profile.
```

Reply:
Tbh I've seen a pretty mixed bag, plenty of references use uppercase for all scattering functions e.g. `I(q) = S(q) * F(q)` since they're all Fourier transforms. I'm fine with switching though.